### PR TITLE
model toggler

### PR DESCRIPTION
**Pull request by iamkallolpratim**
Toggle between gpt 4 & gpt3.5

### CODE REVIEW

File: auth.ts [hunk 1]
```diff
@@ -48,7 +49,7 @@ export const {
       }
 
       if (account?.provider === "google" && profile?.email && profile?.email_verified) {
-        return profile.email_verified && (profile.email.endsWith("@nuclaysolutions.com") || profile.email.endsWith("@givecentral.org") || profile.email.endsWith("@terminal33.io"))
```
Comment (graphicmist) [inline auth.ts:51]: Why are the conditions removed? If you dont want to use them, put them with prod env check

File: app/api/chat/route.ts [hunk 2]
```diff
@@ -5,74 +5,72 @@ import { createClient } from '@supabase/supabase-js'
...
-      },
-      method: 'POST',
-      body: JSON.stringify({
-        message,
-        temperature,
-        model
-      })
-    });
+export async function POST(req: Request) {
+  const selectedModel = window.localStorage.getItem('model')
```
Comment (graphicmist) [inline app/api/chat/route.ts:11]: what happens when no value is in local storage?

File: auth.ts (hunk 1 above)
Comment (iamkallolpratim) [reply auth.ts:51]: doing it

File: app/api/chat/route.ts (hunk 2 above)
Comment (iamkallolpratim) [reply app/api/chat/route.ts:11]: by default it will take 3.5

**Pull request closed without merging**